cmd/server.go

```diff
@@ -55,6 +56,8 @@ sprayproxy server --backend http://localhost:8081 --backend http://localhost:808
 		metricsSrvr.StopServer()
 		return err
 	},
+	// prevent showing usage on RunE return
+	SilenceUsage: true,
```

Comment:
Why do we need this? I don't usually see this with other projects that use cobra.

Reply:
When testing locally, existing on error (not cmd error) was printing the usage which felt odd. Other than that I did not find this affecting affecting cmd. I also found this issue, so it was not only me https://github.com/spf13/cobra/issues/340 . I do not have strong opinion and can revert if you prefer.

Reply:
Let's keep it, but update the comment so the justification is clear:

```suggestion
	// prevent showing usage if RunE returns an error - see https://github.com/spf13/cobra/issues/340
	SilenceUsage: true,
```

Reply:
Updated.